components/TipDetails/index.js

```diff
@@ -1,28 +1,60 @@
 import Image from "next/image";
 import styled from "styled-components";
-import Link from "next/link";
 import { FaChevronLeft } from "react-icons/fa6";
 import { useRouter } from "next/router";
+import Tag from "../Tag";
+import { useState } from "react";
 
 
-export default function TipDetails({ tip }) {
+export default function TipDetails({ tip, plantsToBeTagged }) {
   const router = useRouter();
 
+  const [noRelatedPlants, setNoRelatedPlants] = useState(false);  
+
+  const relatedPlants = tip.relatedPlants.map((relatedPlant) => relatedPlant);
+  const relatedPlantObject = relatedPlants.map((plantID) => plantsToBeTagged.find((plant) => plant.id === plantID));
+  
+  const undefinedCount = relatedPlantObject.filter(plant => plant === undefined).length;
+  
+  const noRelatedPlantsChecker = () => {if (undefinedCount === relatedPlantObject.length) {
+    setNoRelatedPlants(true)
+  }}
+
+
   return (
     <>
       <StyledImageContainer>
             <StyledImage src={tip.imageURL} alt={tip.title} fill />
       </StyledImageContainer>
       
+      <StyledIconContainer onClick={() =>  router.back()} type="button">
+          <FaChevronLeft />
+        </StyledIconContainer>
+
       <StyledTipContainer>
         <StyledH2>{tip.title}</StyledH2>
         <StyledDescription>{tip.bodyContent}</StyledDescription>
+      
+        {!noRelatedPlants ? (<StyledH3>Related Plants</StyledH3>) : ("")}
```

Comment:
Again, just use the logical `&&` operator if no else case exists.

You can also think about combining it with the list below, so you don't have to write two functions for two elements that are visible/hidden based on the same condition.